integration_test/install_test.go

```diff
@@ -216,3 +220,14 @@ func localTestServer() (string, func()) {
 	s := httptest.NewServer(http.FileServer(http.Dir("testdata")))
 	return s.URL, s.Close
 }
+
+func assertManifestPluginIndex(t *testing.T, test *ITest, plugin string) {
+	receiptPath := test.TempDir().Path("receipts/" + plugin + constants.ManifestExtension)
```

Comment:
don't guess filesystem layout; use environment.Paths (unless you're actually testing user-visible behavior, breaking behavior between versions, or Paths type).